node/task.ts

```diff
@@ -1150,42 +1151,77 @@ function _legacyFindFiles_getMatchingItems(
  * @param     path     path to remove
  * @returns   void
  */
-export function rmRF(path: string): void {
-    debug('rm -rf ' + path);
+export function rmRF(inputPath: string): void {
+    debug('rm -rf ' + inputPath);
 
-    // get the lstats in order to workaround a bug in shelljs@0.3.0 where symlinks
-    // with missing targets are not handled correctly by "rm('-rf', path)"
-    let lstats: fs.Stats;
-    try {
-        lstats = fs.lstatSync(path);
-    }
-    catch (err) {
-        // if you try to delete a file that doesn't exist, desired result is achieved
-        // other errors are valid
-        if (err.code == 'ENOENT') {
-            return;
+    if (getPlatform() == Platform.Windows) {
+        // Node doesn't provide a delete operation, only an unlink function. This means that if the file is being used by another
+        // program (e.g. antivirus), it won't be deleted. To address this, we shell out the work to rd/del.
+        try {
+            if (fs.statSync(inputPath).isDirectory()) {
+                debug('removing directory');
```

Comment:
is there a reason to not put the path you are trying to delete in the debug message?

Reply:
Nope, will update